recipes/geofabrics/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "geofabrics" %}
+{% set version = "1.1.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/geofabrics-{{ version }}.tar.gz
+  sha256: 6c14ece1404fadf0e1c5dca710e7d88a9454f0b43abd827cd15684e9bd48878b
+
+build:
+  skip: true
```

Comment:
This won't build anything.

Reply:
Ah thanks - still pretty new to this. I will make your recommended changes. I was getting trouble with linting errors. Perhaps you can give some guidance on that once I approve your suggested changes. I was confused by this: `noarch packages can't have selectors. If the selectors are necessary, please remove noarch: python.` and then also got some build errors after